### PR TITLE
Feat/bypass plugin sync

### PR DESCRIPTION
**Pull request by santokhsingh**

### CODE REVIEW

File: views/design.jade [hunk 1]
```diff
@@ -18,6 +18,6 @@ block content
                     +inputField('glob.company', glob, 'Your Company Name:')
                     +inputField('glob.footer', glob, 'Portal Page Footer:')
             
-            +renderButtons('/design', '/templates', null)
+            +renderButtons('/design', '/templates', '/kongadaptor')
```
Comment (DonMartin76) [inline views/design.jade]: `/kongadapter` please, to keep it consistent.

Comment (DonMartin76) [reply views/design.jade]: It would actually not be correct even

**Review by DonMartin76 — CHANGES_REQUESTED** [review]
Small change in a link there, then it LGTM

File: views/design.jade (hunk 1 above)
Comment (santokhsingh) [reply views/design.jade]: Ah! sorry i thought i modified all the occurrence.

**Review by DonMartin76 — APPROVED** [review]

**Pull request merged**